.circleci/config.yml

```diff
@@ -1750,7 +1750,9 @@ workflows:
 
       - t_ext: *job_ems_compile_ext_colony
 
-      - t_ext: *job_native_test_ext_gnosis
+      # NOTE: We are disabling the gnosis test suite due to version discrepancies that are difficult to fix.
+      #       Check again after (and if) https://github.com/safe-global/safe-contracts/pull/644 is merged.
+      #- t_ext: *job_native_test_ext_gnosis
```

Comment:
You also need to remove it from the `c_ext_benchmarks`: https://github.com/ethereum/solidity/blob/849c2e1092096f04086593235def5a91d1453174/.circleci/config.yml#L1778

Reply:
Done, let's see if that's all.